src/main/groovy/io/seqera/wave/service/blob/BlobCacheService.groovy

```diff
@@ -48,4 +48,17 @@ interface BlobCacheService {
      */
     BlobCacheInfo retrieveBlobCache(RoutePath route, Map<String,List<String>> requestHeaders, Map<String,List<String>> responseHeaders)
 
+    /**
+     * get the size of the blob stored in the cache
+     *
+     * @return {@link Long} the size of the blob stored in the cache
+     */
+    Long getBlobSize(String key)
+
+    /**
+     * delete the blob stored in the cache
+     *
+     */
+    void deleteBlob(String key)
```

Comment:
Are those methods really needed to be exposed in the interface? Likely these operation should only be needed in the service implement

Reply:
yes, these only being used inside service, so no need to expose, I will make the changes

Reply:
done

Reply:
nice. what about adding a unit test?

Reply:
already there
https://github.com/seqeralabs/wave/blob/d62b2ceb1285bac4da40a4a4cc677c4fc955c489/src/test/groovy/io/seqera/wave/service/blob/impl/BlobCacheServiceImplTest.groovy#L88-L131